openmc/cell.py

```diff
@@ -85,6 +85,9 @@ class Cell(IDManagerMixin):
         calculated in a stochastic volume calculation and added via the
         :meth:`Cell.add_volume_information` method. For 'distribmat' cells
         it is the total volume of all instances.
+    volume_std : float
+        Standard deviation in cm^3 of the stochastic volume calculation, added
+        via :meth:`Cell.add_volume_information` method.
```

Comment:
I'm not a huge fan of adding a separate property that basically conveys information related to the volume (which we already have a property for). Given that `ufloat` is already being used elsewhere to indicate uncertainties, the separate property would create an inconsistency across the API in how uncertainties are handled. So, allowing `volume` to be either a plain float or a `uncertainties.ufloat` is certainly cleaner in that regard, but I do see that it would create problems elsewhere. Can you take a shot at trying to use `ufloat` with the `volume` property and we'll see what it looks like?

Reply:
Sure thing- I will look into if/how that can be done.